.gitlab-ci.yml

```diff
@@ -198,6 +199,18 @@ verify:branch:
   script:
     - ASAP_API_URL=$API_URL yarn test:e2e
 
+verify:report:frontend:release:
+  <<: *tmpl_master
+  stage: verify-dev
+  needs:
+    - build:ts
```

Comment:
it doesn't really affect the pipeline but you only really need `deploy:sls:development`